src/ResourceGraph/ResourceGraph/ResourceGraph.format.ps1xml

```diff
@@ -1,30 +1,25 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0"?>
 <Configuration>
   <ViewDefinitions>
     <View>
-      <Name>Microsoft.Azure.Commands.ResourceGraph.Models.PSResourceGraphResponse</Name>
+      <Name>Microsoft.Azure.Commands.ResourceGraph.Models.PSResourceGraphResponse`1[[System.Management.Automation.PSObject]]</Name>
       <ViewSelectedBy>
-        <TypeName>Microsoft.Azure.Commands.ResourceGraph.Models.PSResourceGraphResponse</TypeName>
+        <TypeName>Microsoft.Azure.Commands.ResourceGraph.Models.PSResourceGraphResponse`1[[System.Management.Automation.PSObject]]</TypeName>
       </ViewSelectedBy>
       <ListControl>
         <ListEntries>
           <ListEntry>
             <ListItems>
               <ListItem>
                 <PropertyName>SkipToken</PropertyName>
-                <ItemSelectionCondition>
-                  <ScriptBlock>
-                    $_.SkipToken -ne $null
-                  </ScriptBlock>
-                </ItemSelectionCondition>
```

Comment:
is this removal based on generated ps1xml file.

Reply:
Yes, it is